stock_package_info/views/product_packaging_view.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo>
+    <record id="product_packaging_tree_view" model="ir.ui.view">
+        <field name="name">product.packaging.tree.view</field>
+        <field name="model">product.packaging</field>
+        <field name="arch" type="xml">
+            <tree string="Packaging">
+                <field name="sequence" widget="handle"/>
+                <field name="product_tmpl_id"/>
+                <field name="ean"/>
+                <field name="layer_qty"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="product_packaging_tree_view_product" model="ir.ui.view">
+        <field name="name">product.packaging.tree.view.product</field>
+        <field name="model">product.packaging</field>
+        <field name="arch" type="xml">
+            <tree string="Packaging">
+                <field name="product_pack_tmpl_id"/>
+                <field name="layer_qty"/>
+                <field name="rows"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="product_packaging_form_view" model="ir.ui.view">
```

Comment:
Note actually it is showing up, but not in the `Inventory => Products => [Inventory] Packaging` section